integration-cli/docker_cli_pull_test.go

```diff
@@ -272,6 +272,6 @@ func (s *DockerSuite) TestPullLinuxImageFailsOnWindows(c *check.C) {
 // Regression test for https://github.com/docker/docker/issues/28892
 func (s *DockerSuite) TestPullWindowsImageFailsOnLinux(c *check.C) {
 	testRequires(c, DaemonIsLinux, Network)
-	_, _, err := dockerCmdWithError("pull", "microsoft/nanoserver")
+	_, _, err := dockerCmdWithError("pull", "microsoft/nanoserver:sac2016")
```

Comment:
Arf. there's no `1607` or `ltsc2016` tag 😞 

Should this be using `mcr.microsoft.com/windows/nanoserver` now?

Reply:
The MCR images are all manifest lists so there will be another error message.
The microsoft/nanoserver:sac2016 is a "normal" image that was pushed to Docker Hub.
But it is no longer updated as last update was in October 2018 and nanoserver 2016 is deprecated now.

Maybe we should go with #39096 instead to check for pull error messages from a manifest list?